### ISSUE TITLE
Accessibility - 1010EZ: Remove nested fieldset for better screen reader experience

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#57406]
**Issue description**

The Current compensation from VA screen has nested fieldsets. This isn't a 508 violation, but [nested fieldsets can cause challenges for some screen readers](https://accessibility.blog.gov.uk/2016/07/22/using-the-fieldset-and-legend-elements/). It's best to avoid them when possible. That seems doable here.

![Current compensation from VA screen with red arrows pointing to two fieldsets in HTML markup.](https://images.zenhubusercontent.com/628e7b9093a9366153cb221c/05d3b3b0-35a6-43ff-92a6-eac80e5f55dd)

**Remediation guidance**

Remove top-level fieldset.

**Does this impact visual design?**

No.

**Relevant standards**

N/A

**Comment by hdjustice** [comment]
Need to look into page configuration set up

**Comment by davidakennedy** [comment]
I updated this today to include some of the screens from the new Household section that meet the same criteria. It makes sense to me to bundle this all into one ticket since this is a broad structural change to the markup. I also think it's okay for the Household section to launch as it is with these nested fieldsets because this is more of an enhancement than a bug.
